firebase/functions/src/index.ts

```diff
@@ -355,12 +355,14 @@ exports.addProjectTopicKey = functions.https.onRequest(async (_, res) => {
             res.status(404).send('No projects found');
```

Comment:
We might still need to return from this function.

cc: @frozenhelium

Reply:
Calling `status` will terminate the function. So, we don't need a return here.

Reply:
Does it throw?

Reply:
done